auth-web/src/components/auth/manage-business/CreateNewBusinessHelp.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div id="help-resolution" class="mb-12">
+    <div class="ma-6">
+      <div class="help-btn top" @click="helpToggle = !helpToggle">
+        <v-icon color="primary">mdi-help-circle-outline</v-icon>
+        <span v-if="!helpToggle" class="pl-2">{{ helpContent.toggleHelpShow }}</span>
+        <span v-else class="pl-2">{{ helpContent.toggleHelpHide }}</span>
+      </div>
+      <v-expand-transition v-if="helpToggle">
+        <section class="help-section info-text">
+          <header class="help-header mt-4">
+            <p class="title-text mt-4">{{ helpContent.title }}</p>
+          </header>
+          <br>
+          <div class="help-text mt-4">
+            <p>{{ helpContent.intro }}</p>
```

Comment:
As written, this is not a general-purpose help component -- it's pretty specific to the layout (title, intro, steps).

I wonder if we could make this general purpose by importing a HTML block instead of a JSON file?

Reply:
An even better idea: make the help content its own component, and use this "help wrapper" to show the buttons and conditionally show the help content component.

Reply:
So it would look like this:
```
<HelpComponent>
   <template #content>
      <CreateNewBusines />
   </template>
</HelpComponent>
```

Reply:
Please reach out to me ASAP if you don't understand this or totally disagree with it. Otherwise, I'm happy to continue the discussion here.

Reply:
Thanks. Much better.